lib/ansible/modules/cloud/amazon/lambda_policy.py

```diff
@@ -110,7 +110,8 @@
       statement_id: lambda-s3-myBucket-create-data-log
       action: lambda:InvokeFunction
       principal: s3.amazonaws.com
-      source_arn: arn:aws:s3:eu-central-1:123456789012:bucketName
+      source_arn: arn:aws:s3:::bucketName
```

Comment:
We have tests that run in CI that specify a complete ARN: https://github.com/ansible/ansible/blob/a8d4bf86421d151d8df7132e8e87d04b6662f45a/test/integration/targets/lambda_policy/tasks/main.yml#L166-L184

Perhaps there is another reason your task failed? It would be fine to also document an abbreviated ARN example, but I don't think we should overwrite an example using a full ARN.

Reply:
Or adding another example using a full ARN that works would be fine.